lib/LastLoginDate.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * ownCloud
+ *
+ * @author Hari Bhandari <haribhandari07@gmail.com>
+ * @copyright Copyright (c) 2020 Hari Bhandari haribhandari07@gmail.com
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\Testing;
+
+use OC\User\AccountMapper;
+use OCP\IRequest;
+use OC\OCS\Result;
+
+class LastLoginDate {
+	/**
+	 * @var AccountMapper
+	 */
+	private $accountMapper;
+
+	/**
+	 * @var IRequest
+	 */
+	private $request;
+
+	/**
+	 *
+	 * @param AccountMapper $accountMapper
+	 * @param IRequest $request
+	 *
+	 * @return void
+	 */
+	public function __construct(
+		AccountMapper $accountMapper, IRequest $request
+	) {
+		$this->accountMapper = $accountMapper;
+		$this->request = $request;
+	}
+
+	public function setLastLoginDate($param) {
+		$user = $param['user'];
+		$account = $this->accountMapper->getByUid($user);
```

Comment:
we will need to handle the case if the user does not exist, catch it and return some 4xx code.